e2e/_suites/fleet/features/fleet_mode_agent.feature

```diff
@@ -26,6 +26,17 @@ Examples:
 | centos | tar       |
 | debian | tar       |
 
+@upgrade-agent
+Scenario Outline: Upgrading the installed <os> agent
```

Comment:
I have a PR coming https://github.com/elastic/e2e-testing/pull/439 that will update the whole file so linting passes.  you'll find that you can't use the | tar | usage phrasing as it is the only variable used.  Just replace it in the Gherkin with "tar".

Reply:
let push your issue i deal with conflicts and update file afterwards

Reply:
done